src/main/java/com/rakovets/course/java/core/practice/array/Task01.java

```diff
@@ -34,7 +35,11 @@ public static void main(String[] args) {
     static double getAverageMark(int[] marks) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0;
+        double sum=0;
+        for (int i=0; i<marks.length; i++){
+        sum=sum+marks[i];
+        }
```

Comment:
Violation of Code Convention.
```suggestion
        for (int i = 0; i < marks.length; i++) {
            sum = sum + marks[i];
        }
```